### PR TITLE
fix: #360 - return void instead of promise in preload.mediadevices-shim

### LINKED ISSUE #360: preload.mediadevices-shim: Promise returned in function argument where a void return was expected.
## Description

The following code has been marked as a [bug](https://sonarcloud.io/project/issues?open=AYh_YKsnW2wJeQETzpxS&id=manusa_electronim):

https://github.com/manusa/electronim/blob/0ad1c686c3461dbec38bd6aa1bb2feae3c86857a/src/tab-manager/preload.mediadevices-shim.js#L148-L151

It should be changed to:

```javascript
  const updateSourcesFunction = () => {
    desktopCapturer.getSources(DEFAULT_SOURCES_OPTIONS).then(setSources);
  };
  useEffect(() => {
    setTimeout(updateSourcesFunction, sources ? 300 : 0);
  }, [sources]);
```

Please **note** that for the tests to pass correctly, the following lines:

https://github.com/manusa/electronim/blob/0ad1c686c3461dbec38bd6aa1bb2feae3c86857a/src/tab-manager/preload.mediadevices-shim.js#L27-L30

need to be changed to:

```javascript
const desktopCapturer = {
  // eslint-disable-next-line no-undef
  getSources: async opts => ipcRenderer.invoke(APP_EVENTS.desktopCapturerGetSources, opts)
};
```

### PR DESCRIPTION
**Pull request by vishnukaushik**
Fixes #360 
Changed the code to return void instead of Promise in preload.mediadevices-shim

### CODE REVIEW

**Comment by vishnukaushik** [comment]
@manusa Could you please take a look at this PR when you get a chance?
I need to get maintainers' approval to get the workflow started.

**Review by manusa — APPROVED** [review]
LGTM, thx!

**Pull request merged**
